src/app/components/card/card.component.css

```diff
@@ -0,0 +1,138 @@
+.card {
+    margin-top: 4rem;
+}
+
+.card__content {
+    background-color: var(--white);
+    border-radius: 1rem;
+    border: var(--white) solid 2px;
+    box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, 0.15);
+    width: 100%;
+    padding: 1rem;
+    position: relative;
+    transition: border 0.2s ease-in-out;
+}
+
+.card__content:hover {
+    cursor: pointer;
+    border: var(--gray) solid 2px;
+}
+
+.card__content:hover img {
+    transition: transform 0.2s ease-in-out;
+    transform: scale(1.2);
+}
```

Comment:
con SCSS (SASS) puoi compattare questo codice

come mai usi l'underscore? che tipo di standard è? ho sempre usato dash (-) per i nomi delle classi

Reply:
È BEM. È il tipo di notazione che preferisco e a cui sono abituata. Sul SASS rende meglio ma considerando che non ci sono troppi componenti e che il focus principale del progetto è lavorare in Angular (pensavo di sostituire il CSS con i componenti material quando avrò imparato un minimo, giusto per sovrapporre troppe cose sin da subito) ho lasciato il CSS.

La notazione BEM ha questa struttura:

`componente-principale
componente-principale__componente-interno
componente-principale__componente-interno--modificatore
`